src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/CreateNewUserWithRolesTest.java

```diff
@@ -155,7 +135,7 @@ public void should_return_404_when_adding_new_user_to_an_pending_organisation()
 
         userProfileCreateUserWireMock(HttpStatus.CREATED);
         Map<String, Object> newUserResponse =
-                professionalReferenceDataClient.addUserToOrganisation(orgIdentifierResponse, userCreationRequest, hmctsAdmin);
+                professionalReferenceDataClient.addUserToOrganisation(orgIdentifierResponse, userCreationRequest, hmctsAdmin, null);

```

Comment:
why we are explicitly passing null? can't we create separate method?

Reply:
This has been addressed in this commit: https://github.com/hmcts/rd-professional-api/pull/492/commits/c46fbf238200ba333c186b8c74cb4dd622bd2e4c